themes/src/main/resources/theme/base/login/login-password.ftl

```diff
@@ -10,10 +10,17 @@
                     <div class="${properties.kcFormGroupClass!} no-bottom-margin">
                         <hr/>
                         <label for="password" class="${properties.kcLabelClass!}">${msg("password")}</label>
-                        <input tabindex="2" id="password" class="${properties.kcInputClass!}" name="password"
-                               type="password" autocomplete="on" autofocus
-                               aria-invalid="<#if messagesPerField.existsError('password')>true</#if>"
-                        />
+                        <div class="${properties.kcInputGroup!}">
```

Comment:
As this is a widely used user-fancy UI, do we want to include release notes about migrating themes and the new option introduced? AFAIK, we don't recommend customizing templated directly, so I'm not 100% sure.

Also, can we assume all deployments want the new behavior? Otherwise, we might think about adding a realm setting?

The main point for above is that those who don't want the new behavior now need to customize the template directly.

Reply:
Have not thought about that but I got your point. Nevertheless, I am of the opinion that this realm settings you are talking about are some kind of feature flags; something like user-registration, forget-password and things like that. I wont describe my PR as a feature but rather a convenient quality-of-life improvement for the user - which would find its way into the UI (hopefully?) anyway when we upgrade to newer PatternFly versions, right? Of course you could also hide it behind such a feature flag but doing this for usability-improvements did not feel quite right to me.

Thanks for bringing this up @pedroigor! :) Very open to discussions, any other opinions?

Reply:
I see well-known IdPs doing the same. I'm just not sure if we should consider this change backward compatible because people might not expect the new behavior for whatever reason such as compliance with the company UX policies or even for security reasons.

We should definitely hear other opinions. @stianst @mposolda Any thoughts?

Reply:
I think we can live without backwards compatibility flag. At least for now. Maybe if people complaints, we can add backwards compatibility flag later. Some major IDPs like Google, Facebook, Twitter are also doing the same, so looks like a way forward.

But will be good to document this in the migration guide as people, who have their own templates for login themes, might be definitely affected by this. I guess we target this for Keycloak 23 (also considering backwards compatibility etc, I personally vote for rather not adding it to 22.0.X). In that case, change might need to be documented in the file `changes_23-0-0.adoc` ?

Reply:
I agree, documented the changes in `changes_23-0-0.adoc`. Could you have a look?